src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -152,6 +152,155 @@
           text: >
             az account lock update --name lockName --notes newNotesHere --lock-type CanNotDelete
     """
+helps['account management-group'] = """
+    type: group
+    short-summary: Manage Azure Management Groups.
+"""
+
+helps['account management-group subscription'] = """
+    type: group
+    short-summary: Subscription operations for Management Groups.
+"""
+
+helps['account management-group list'] = """
+    type: command
+    short-summary: List all management groups.
+    long-summary: List of all management groups in the current tenant.
+    examples:
+        - name: List all management groups
+          text: >
+             az account management-group list
+"""
+
+helps['account management-group show'] = """
+    type: command
+    short-summary: Get a specific management group.
+    long-summary: Get the details of the management group.
+    parameters:
+        - name: --name -n
+          type: string
+          short-summary: Name of the management group.
+        - name: --expand -e
+          type: bool
+          short-summary: If given, lists the children in the first level of hierarchy.
+        - name: --recurse -r
+          type: bool
+          short-summary: If given, lists the children in all levels of hierarchy.
+    examples:
+        - name: Get a management group.
+          text: >
+             az account management-group show --name GroupName
+        - name: Get a management group with children in the first level of hierarchy.
+          text: >
+             az account management-group show --name GroupName -e
+        - name: Get a management group with children in all levels of hierarchy.
+          text: >
+             az account management-group show --name GroupName -e -r
+"""
+
+helps['account management-group create'] = """
+    type: command
+    short-summary: Create a new management group.
+    long-summary: Create a new management group.
```

Comment:
`long-summary` should provide additional information, instead of being a duplicate of `short-summary`.